src/store/global/global.reducer.js

```diff
@@ -43,6 +43,12 @@ function getInitialGlobalState () {
     nextForgers: [],
     coordinatorStateTask: {
       status: 'pending'
+    },
+    estimatedRewardTask: {
```

Comment:
I would encapsulate both `estimatedRewardTask` and `earnedRewardTask` inside an `airdop` parent:

```js
airdrop: {
  estimatedRewardTask
  earnedRewardTask
}
```

Reply:
airdrop => reward

Reply:
Yep, and maybe in plural: `rewards` instead of `airdrop`